stylus/include/Home.styl

```diff
@@ -17,16 +17,17 @@
 		padding-bottom: 3rem
 	>div:nth-child(5), div:nth-child(6)
 		display: flex
-		justify-content: center
+		justify-content: space-evenly
 		padding-bottom: 1rem
+		padding-left: 5rem
+		padding-right: 5rem
 		>div
+			background-color: $Mustard
 			position: relative
 			>img
 				height: 20rem
 				position: relative
 				width: 25rem
```

Comment:
Remove the width here. It will mess with the flex.

Reply:
When I remove the `width` property the flexbox tries to fit all of the images in one row and they end up smushed together.  I'm still learning all the ins and outs of flexbox; is there some trick I can use to ensure the images are not squished horizontally?

Reply:
Remove the height as well.  Our current practice is to use images with height / width ratios that are all similar.

Also, check out `flex-basis`

Reply:
Note how as it stands without the `height` style, some of the images have the gold background coming through because they're not as tall as the tallest image in the row. It doesn't look quite right that way. That's why I suggested either keeping height and using media queries or cropping the images. I'll let @cbaron make the call there.